### ISSUE TITLE
Title: Question about class name in trainer.py's _save_checkpoint method

### ISSUE THREAD

**Issue by Dasol-Choi** [2U1/Pixtral-Finetune#1]
Thank you for sharing a great project!

While running the code, I noticed a potential typo or inconsistency in the `trainer.py file`. In the `_save_checkpoint method`, the `super()` call references `LLamaVTrainer`, but the class is named `PixtralTrainer`. 
```
def _save_checkpoint(self, model, trial, metrics=None):
    ...
    else:
        super(LLamaVTrainer, self)._save_checkpoint(model, trial, metrics)
```

Should this be updated to `super(PixtralTrainer, self)` for consistency with the class name? I’d appreciate clarification on whether this is intentional or requires a fix.

**Comment by 2U1** [comment]
@Dasol-Choi Yes you are right. I was changing the code based on the llama3.2-vision finetuning code. I missed changing it.
I'll fix it right away. 

Thanks for letting me know.
